specification/batch/resource-manager/Microsoft.Batch/stable/2021-06-01/examples/BatchAccountListOutboundNetworkDependenciesEndpoints.json

```diff
@@ -0,0 +1,72 @@
+{
+  "parameters": {
+    "accountName": "sampleacct",
+    "resourceGroupName": "default-azurebatch-japaneast",
+    "api-version": "2021-06-01",
+    "subscriptionId": "subid"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "category": "Azure Batch",
+            "endpoints": [
+              {
+                "domainName": "japaneast.batch.azure.com",
+                "description": "Applicable to all Azure Batch pools.",
+                "endpointDetails": [
+                  {
+                    "port": 443
+                  }
+                ]
+              }
+            ]
+          },
```

Comment:
This resource is not defined as per ARM contract.
Any ARM resource should contain id/name/type as top level properties and RP specific properties should be defined under properties envelope.

Refer [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope)

Reply:
OutboundNetworkDependenciesEndpoints are not a true customer-owned ARM resource and do not have ids, names, tags etc. This endpoint gives some read-only information about batchAccounts and does not represent a sub-resource of batchAccounts.

This endpoint is a widespread Vanguard requirement made according to a Networking spec. It should be noted that many teams are implementing endpoints exactly like this one and some teams have already gotten them approved+merged+released.
See AKS at https://github.com/Azure/azure-rest-api-specs/pull/14147/files and a similar dispute at https://github.com/Azure/azure-rest-api-specs/pull/14332#discussion_r629606824

Reply:
Alright, sounds good. Signing off as the same pattern exists across other services.